lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -84,6 +89,7 @@ class CircleImageBase extends NodeBase {
       this.width = width;
       this.height = height;
       this.radius = 0.5 * this.width;
+      this.refreshNeeded = false;
```

Comment:
Are you sure we don't need the refresh here because of the hover/selected state? I'm not sure.